terraso_backend/apps/project_management/models/site_notes.py

```diff
@@ -0,0 +1,32 @@
+# Copyright © 2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+from django.db import models
+
+from apps.core.models import User
+
+
+class SiteNote(models.Model):
+    site = models.ForeignKey("Site", on_delete=models.CASCADE, related_name="notes")
```

Comment:
question: can we import `Site` and use that? Or does it cause a circular dependency?

Reply:
Importing Site causes a circular dependency, unfortunately.